packages/grafana-ui/src/components/Combobox/getMultiComboboxStyles.ts

```diff
@@ -25,6 +25,8 @@ export const getMultiComboboxStyles = (theme: GrafanaTheme2) => {
       border: 'none',
       outline: 'none',
       background: 'transparent',
+      width: '4px',
+      pointerEvents: 'none',
```

Comment:
This is causing tests to fail, and might not be good a11y practice. Also, why is the `4px` thing working? I'd expect it to grow regardless, and in the devtools it seems like it takes up almost the same space

Reply:
Good call about both of these 🙌  I have reworked what I was trying to do, which was to be able to get the whole container clickable to open the options instead of only by clicking the input.